x-pack/plugins/apm/public/index.js

```diff
@@ -28,6 +30,25 @@ import { history } from './components/shared/Links/url_helpers';
 
 import { I18nContext } from 'ui/i18n';
 
+// render APM feedback link in global help menu
+chrome.helpExtension.set(domElement => {
+  ReactDOM.render(
+    <EuiLink
+      href="https://discuss.elastic.co/c/apm"
+      target="_blank"
+      rel="noopener"
+    >
+      {i18n.translate('xpack.apm.feedbackMenu.provideFeedbackTitle', {
+        defaultMessage: 'Give APM Feedback'
+      })}
+    </EuiLink>,
+    domElement
```

Comment:
Can we add another link to the Kibana Migration Assistant? Link is `/app/kibana#/management/elasticsearch/upgrade_assistant` and happens within Kibana so no `target` and no `rel` props necessary, afaik?